src-plugins/medVtkView/medVtkView.cpp

```diff
@@ -552,16 +552,12 @@ void medVtkView::resetKeyboardInteractionModifier()
     d->rubberBandZoomParameter->setValue(false);
 }
 
-void medVtkView::showHistogram(bool checked)
+void medVtkView::showHistogram()
 {
-    if (!checked)
+    if (d->transFun != NULL )
     {
-        if (d->transFun !=NULL )
-        {
-            delete d->transFun ;
-            d->transFun=NULL;
-        }
-        return;
```

Comment:
In fact, there is no more boolean checking, so, this `if` condition is no longer necessary